front/src/common/BootstrapSNCF/MultiSelectSNCF.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+
+type MultiSelectSNCFProps = {
+  multiSelectTitle: string;
+  multiSelectSubTitle?: string;
+  selectOptions: { label: string }[];
```

Comment:
```suggestion
  options: string[];
```
I think options is okay

Reply:
Done